packages/jest-types/src/Global.ts

```diff
@@ -27,6 +27,12 @@ export type GeneratorReturningTestFn = (
   this: TestContext | undefined,
 ) => TestReturnValueGenerator;
 
+export interface FunctionLike {
+  readonly name: string;
+}
```

Comment:
I get the idea, but this also allows some object with `name` prop.

Reply:
Yeah, this was to maintain parity with `@types/jest`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e51f6d661ae12f664def25a3019304be41bd2d9c/types/jest/index.d.ts#L385

...but now that IE 11 support isn't necessary it does make sense to take it out. 👍

Reply:
Parity with `@types/jest` is not a goal in and of itself, fwiw 🙂